src/main/resources/db/migration/V1__init.sql

```diff
@@ -79,57 +95,94 @@ create table if not exists bookcrossing.t_user (
     primary key (user_id)
 );
 
+comment on table bookcrossing.t_user is 'Basic user data';
+
 create table if not exists bookcrossing.t_user_role (
     user_id integer not null,
     role_id integer not null,
     primary key (user_id, role_id)
 );
 
-alter table bookcrossing.t_action_mail_user
+comment on table bookcrossing.t_user_role is 'Table of user and role accordance';
+
+alter table if exists bookcrossing.t_action_mail_user
     add constraint action_foreign_user
         foreign key (user_id) references bookcrossing.t_user (user_id) ON DELETE CASCADE;
 
-alter table bookcrossing.t_attach
+create index if not exists action_foreign_user on bookcrossing.t_action_mail_user (user_id)
+    where user_id is not null;
```

Comment:
In table `user_id integer not null,`
So clause  `where user_id is not null;` is absolutely unnecessary

Reply:
Please check similar cases below